Image building fails with `files/usr/bin/skopeo: not found`

With the current `staging` HEAD building image fails with error: `/bin/sh: 1: files/usr/bin/skopeo: not found`.

I think this was introduced after in https://github.com/confidential-containers/cloud-api-adaptor/pull/263.

### Reproduce

```bash
git clone https://github.com/kata-containers/kata-containers -b CCv0
git clone https://github.com/confidential-containers/cloud-api-adaptor.git

cd cloud-api-adaptor/azure/image/
export CLOUD_PROVIDER=azure
```

```console
# make build
cd "../.." && make agent-protocol-forwarder
make[1]: Entering directory '/root/foobar/cloud-api-adaptor'
CGO_ENABLED=0 go build -tags=azure -o "agent-protocol-forwarder" "cmd/agent-protocol-forwarder/main.go"
...
...
   Compiling oci-distribution v0.8.1 (https://github.com/krustlet/oci-distribution?rev=1ba0d94a900a97aa1bcac032a67ea23766bcfdef#1ba0d94a)
   Compiling image-rs v0.1.0 (https://github.com/confidential-containers/image-rs?rev=a1d7ba31201d9d7a575d05c5fed1f2cb2142a842#a1d7ba31)
   Compiling kata-agent v0.1.0 (/root/foobar/kata-containers/src/agent)
    Finished release [optimized] target(s) in 5m 44s
make[1]: Leaving directory '/root/foobar/kata-containers/src/agent'
install -D --compare "../../../kata-containers/src/agent/target/x86_64-unknown-linux-musl/release/kata-agent" "files/usr/local/bin/kata-agent"
files/usr/bin/skopeo --policy "files/etc/containers/policy.json" copy "docker://k8s.gcr.io/pause:3.6" "oci:pause:3.6"
/bin/sh: 1: files/usr/bin/skopeo: not found
make: *** [Makefile:117: pause] Error 127
```

Same can be reproduced for `aws`.

--- a/aws/image/Makefile
+++ b/aws/image/Makefile
@@ -111,11 +111,11 @@ $(UMOCI_SRC)/umoci: $(UMOCI_SRC)
 $(UMOCI): $(UMOCI_SRC)/umoci
 	install -D --compare "$(UMOCI_SRC)/umoci" "$@"
 
-$(PAUSE_SRC):
-	$(SKOPEO) --policy "$(FILES_DIR)/etc/containers/policy.json" copy "$(PAUSE_REPO):$(PAUSE_VERSION)" "oci:$(PAUSE_SRC):$(PAUSE_VERSION)"
+$(PAUSE_SRC): $(SKOPEO_SRC)/bin/skopeo
+	$(SKOPEO_SRC)/bin/skopeo --policy "$(FILES_DIR)/etc/containers/policy.json" copy "$(PAUSE_REPO):$(PAUSE_VERSION)" "oci:$(PAUSE_SRC):$(PAUSE_VERSION)"
 
-$(PAUSE): | $(PAUSE_SRC)
-	$(UMOCI) unpack --rootless --image "$(PAUSE_SRC):$(PAUSE_VERSION)" "${FILES_DIR}/$(PAUSE_BUNDLE)"
+$(PAUSE): | $(PAUSE_SRC) $(UMOCI_SRC)/umoci
+	$(UMOCI_SRC)/umoci unpack --rootless --image "$(PAUSE_SRC):$(PAUSE_VERSION)" "${FILES_DIR}/$(PAUSE_BUNDLE)"
 
 clean:
 	rm -f "$(IMAGE_FILE)" "$(UBUNTU_IMAGE_FILE)" $(BINARIES)

--- a/azure/image/Makefile
+++ b/azure/image/Makefile
@@ -113,11 +113,11 @@ $(UMOCI_SRC)/umoci: $(UMOCI_SRC)
 $(UMOCI): $(UMOCI_SRC)/umoci
 	install -D --compare "$(UMOCI_SRC)/umoci" "$@"
 
-$(PAUSE_SRC):
-	$(SKOPEO) --policy "$(FILES_DIR)/etc/containers/policy.json" copy "$(PAUSE_REPO):$(PAUSE_VERSION)" "oci:$(PAUSE_SRC):$(PAUSE_VERSION)"
+$(PAUSE_SRC): $(SKOPEO_SRC)/bin/skopeo
+	$(SKOPEO_SRC)/bin/skopeo --policy "$(FILES_DIR)/etc/containers/policy.json" copy "$(PAUSE_REPO):$(PAUSE_VERSION)" "oci:$(PAUSE_SRC):$(PAUSE_VERSION)"
 
-$(PAUSE): | $(PAUSE_SRC)
-	$(UMOCI) unpack --rootless --image "$(PAUSE_SRC):$(PAUSE_VERSION)" "${FILES_DIR}/$(PAUSE_BUNDLE)"
+$(PAUSE): | $(PAUSE_SRC) $(UMOCI_SRC)/umoci
+	$(UMOCI_SRC)/umoci unpack --rootless --image "$(PAUSE_SRC):$(PAUSE_VERSION)" "${FILES_DIR}/$(PAUSE_BUNDLE)"
 
 clean:
 	rm -f "$(IMAGE_FILE)" "$(UBUNTU_IMAGE_FILE)" $(BINARIES)

--- a/libvirt/image/Makefile
+++ b/libvirt/image/Makefile
@@ -114,11 +114,11 @@ $(UMOCI_SRC)/umoci: $(UMOCI_SRC)
 $(UMOCI): $(UMOCI_SRC)/umoci
 	install -D --compare "$(UMOCI_SRC)/umoci" "$@"
 
-$(PAUSE_SRC):
-	$(SKOPEO) --policy "$(FILES_DIR)/etc/containers/policy.json" copy "$(PAUSE_REPO):$(PAUSE_VERSION)" "oci:$(PAUSE_SRC):$(PAUSE_VERSION)"
+$(PAUSE_SRC): $(SKOPEO_SRC)/bin/skopeo
+	$(SKOPEO_SRC)/bin/skopeo --policy "$(FILES_DIR)/etc/containers/policy.json" copy "$(PAUSE_REPO):$(PAUSE_VERSION)" "oci:$(PAUSE_SRC):$(PAUSE_VERSION)"
 
-$(PAUSE): | $(PAUSE_SRC)
-	$(UMOCI) unpack --rootless --image "$(PAUSE_SRC):$(PAUSE_VERSION)" "${FILES_DIR}/$(PAUSE_BUNDLE)"
+$(PAUSE): | $(PAUSE_SRC) $(UMOCI_SRC)/umoci
+	$(UMOCI_SRC)/umoci unpack --rootless --image "$(PAUSE_SRC):$(PAUSE_VERSION)" "${FILES_DIR}/$(PAUSE_BUNDLE)"
 
 clean:
 	rm -f "$(IMAGE_FILE)" "$(UBUNTU_IMAGE_FILE)" $(BINARIES)

